ubiquibot-config-default.json

```
@@ -90,5 +90,14 @@
   "enable-access-control": {
     "label": false,
     "organization": true
+  },
+  "bounty-rewards-cap": {
+    "issue_issuer": true,
+    "issue_collaborator": true,
+    "issue_assignee": true,
+    "issue_default": true,
+    "review_issuer": true,
+    "review_collaborator": true,
+    "review_default": true
```

it should be in kebab-case
```suggestion
    "review-default": true
```
it will be changed to camelCase by #711 but I'm not sure how long it will take